### ISSUE TITLE
SSD + ResNet

### ISSUE THREAD

**Issue by David-Jin** [weiliu89/caffe#130]
@weiliu89 
Hi, I know you just added scripts for training ResNet under SSD branch.
1. Does this SSD's code support of trainning ResNet?  If it does, where did you realized it? in which file?
2. The script "ssd_pascal_resnet.py" is for training ResNet under SSD, but I can't find any code to demo the ResNet's result like "ssd_pascal_video.py".
3. The mAP is just 73%(Given by you), it is seems not very accuracy than the VGG network. Is it Normal?
Can you give me any advice? 
Thank you very much.

**Comment by weiliu89** [comment]
You could adjust according to various of scripts I provided.

I personally don't see strong evidence of the advantage of ResNet over VGGNet in the SSD framework.

**Comment by XHUJOY** [comment]
@David-Jin Maybe, because VGG16 is pretrained on the ILSVRC CLS-LOC dataset, but ResNet is not pretrained on the ILSVRC CLS-LOC dataset. @weiliu89 Did you test the map of ssd model using VGG16 pretrained only  imagenet cls task dataset?

**Comment by weiliu89** [comment]
@XHUJOY ResNet has shown some advantage over VGGNet under Faster R-CNN framework. Maybe I didn't use ResNet in the most optimal way, or maybe my (reduced) VGGNet baseline is better than the one which is used in Faster R-CNN.

**Comment by XHUJOY** [comment]
is the reduced VGGNet trained by youself? I can't find the  paper about the model and i know deeplab use   the same arch which initted by gaussian random value.

**Comment by weiliu89** [comment]
@XHUJOY I manually subsampled parameters from fc6 and fc7 of the original VGGNet. I don't think deeplab is trained from scratch.

**Comment by borissherman** [comment]
Hi, could you please comment on SSD runtime with ResNet-101 compared to VGG-16 reduced. 
In theory ResNet should be faster with less Flops, but in my runs I see x3 slower performance. 
Using GTX 1080 with cuda7.5+cudnn v5 on 300x300 input.

**Comment by zt706** [comment]
@borissherman in my experience, the GTX 1080 need cuda8.0

**Comment by borissherman** [comment]
@zt706 Yes, thanks, found that also, x1.5 improvement with cuda 8.0 on GTX 1080.

**Comment by birdwcp** [comment]
I also don't see strong evidence of the advantage of ResNet50 over VGGNet16 ,maybe VGGNet is enough for detection mission.     darknet19 is even more simple than VGGNet
